packages/components/src/internal/components/domainproperties/DomainForm.tsx

```diff
@@ -1179,73 +1196,113 @@ export class DomainFormImpl extends React.PureComponent<IDomainFormInput, IDomai
-                                                        getDomainFields={this.getDomainFields}
-                                                        fieldDetailsInfo={fieldDetails.detailsInfo}
-                                                        domainIndex={domainIndex}
-                                                        index={i}
-                                                        expanded={expandedRowIndex === i}
-                                                        expandTransition={expandTransition}
-                                                        onChange={this.onFieldsChange}
-                                                        onExpand={this.onFieldExpandToggle}
-                                                        onDelete={this.onDeleteField}
-                                                        maxPhiLevel={maxPhiLevel}
-                                                        dragging={dragId === i}
-                                                        availableTypes={availableTypes}
-                                                        showDefaultValueSettings={domain.showDefaultValueSettings}
-                                                        defaultDefaultValueType={domain.defaultDefaultValueType}
-                                                        defaultValueOptions={domain.defaultValueOptions}
-                                                        appPropertiesOnly={appPropertiesOnly}
-                                                        showFilePropertyType={showFilePropertyType}
-                                                        successBsStyle={successBsStyle}
-                                                        isDragDisabled={
-                                                            filtered || domainFormDisplayOptions.isDragDisabled
-                                                        }
-                                                        domainFormDisplayOptions={domainFormDisplayOptions}
-                                                    />
-                                                );
-                                            })}
-                                            {provided.placeholder}
-                                        </Form>
-                                    </div>
-                                )}
-                            </Droppable>
-                        </StickyContainer>
-                    </DragDropContext>
+                    summaryViewMode ? (
+                        <DomainPropertiesGrid
+                            domain={domain}
+                            search={search}
+                            selectAll={selectAll}
+                            actions={actions}
+                            appPropertiesOnly={appPropertiesOnly}
+                        />
+                    ) : (
```

Comment:
(Ignorable side-comment) I personally liked the indenting of the nested ternary pre-lint, but I'll follow the linter's opinions